src/lib/final_report.ml

```diff
@@ -417,6 +443,11 @@ module To_workflow
             let wf = get_vcf repr in
             let title = sprintf "VCF: %s" name in
             title, Some wf#product#path)
+        @ (Option.value_map ~default:(["Kallisto", None]) kallisto ~f:(fun ks ->
```

Comment:
Hmm, I think you are right about the empty list default; but I don't think I understand your suggestion in the second comment. Doesn't `other_resultes_section` is a list of (string, path)? If so, I don't think `["Kallisto"; ["title", path; ...]]` should fit there, right?